c/cytoolz/build_info.json

```diff
@@ -1,17 +1,17 @@
 {
-  "maintainer": "AbhiIBM",
+  "maintainer": "aastha-sharma2",
   "package_name": "cytoolz",
   "github_url": "https://github.com/pytoolz/cytoolz.git",
-  "version": "0.12.2",
+  "version": "0.12.3",
+  "wheel_build" : "true",
   "default_branch": "master",
   "package_dir": "c/cytoolz",
   "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
-  "build_script": "cytoolz_ubi_8.7.sh",
+  "build_script": "cytoolz_ubi_9.3.sh",
   "validate_build_script": "true",
   "use_non_root_user": "false",
   "docker_build": "true",
-  "0.12.*": {
-    "dir": "ubi8.7",
-    "build_script": "cytoolz_ubi_8.7.sh"
+  "*": {
```

Comment:
@nikhil-kalbande not sure if these is correct, for ubi8.7 we had dockerfile dir and also I see "docker_cmd" attr here. Need help to validate this

Reply:
@aastha-sharma2  yes we will need to add dockerfile here. For now marl docker_build: false.. Take help from @vinodk99 to raise another PR to add dockerfile also